hera_qm/utils.py

```diff
@@ -138,3 +140,79 @@ def generate_fullpol_file_list(files, pol_list):
                 file_list.append(jd_list)
 
     return file_list
+
+
+def metrics2mc(filename, ftype):
+    """ Read in file containing quality metrics and stuff into a dictionary which
```

Comment:
Could we document how to add a new value to the dictionary we pass to M&C? It's not totally clear to me if it's enough to add it to the metrics dictionary or whether the module-level dictionaries need to be edited as well. Perhaps we can add it to the README? Or write a memo?